### PR TITLE
quick_fix_for_type_1

### PR DESCRIPTION
**Pull request by kulrk**
Signed-off-by: kulrk <rohan.1.kulkarni@seagate.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: libs/iostability/iostability_lib.py [hunk 1]
```diff
@@ -51,6 +51,19 @@ def __init__(self, max_retries, timeout, access_key=ACCESS_KEY,
         self.max_retries = max_retries
         self.http_client_timeout = timeout
 
+    def get_s3bench_logfile_label_from_test_log_file(self, test_log_file):
```
Comment (priyankaborawake) [inline libs/iostability/iostability_lib.py:54]: Can this be moved to s3bench lib ?

File: libs/iostability/iostability_lib.py [hunk 2]
```diff
@@ -80,19 +93,71 @@ def execute_workload_distribution(self, distribution, clients, total_obj,
                 resp = s3bench.s3bench(ACCESS_KEY, SECRET_KEY, bucket=bucket_name,
                                        num_clients=cur_clients, num_sample=samples,
                                        obj_name_pref="object-", obj_size=size,
-                                       skip_cleanup=skip_cleanup, duration=None,
+                                       duration=None,
```
Comment (priyankaborawake) [inline libs/iostability/iostability_lib.py:96]: Please add logging before start of each operation.

File: libs/iostability/iostability_lib.py [hunk 3]
```diff
@@ -80,19 +93,71 @@ def execute_workload_distribution(self, distribution, clients, total_obj,
...
+                                       httpclientimeout=self.http_client_timeout,
+                                       skip_write=True,
+                                       skip_cleanup=True,
+                                       skip_read=False,
+                                       validate=True
+                                       )
+                self.log.info("Loop: %s Workload: %s objects of %s with %s parallel clients.",
+                              loop, samples, size, clients)
+                self.log.info("Log Path %s", resp[1])
+                assert not s3bench.check_log_file_error(resp[1]), \
```
Comment (priyankaborawake) [inline libs/iostability/iostability_lib.py:132]: Please add which operation failed in assert statement

**Review by priyankaborawake — CHANGES_REQUESTED** [review]
@kulrk Added minor comments. 
Please check.

**Comment by shailesh-vaidya** [comment]
Closing as an obsolete

**Pull request closed without merging**